core/src/test/java/org/lint/azzert/LintAssertTest.java

```diff
@@ -46,4 +46,44 @@ void azzert() throws Exception {
         Assertions.assertEquals(9, assertsInMethod.apply(methods, "azzert"));
     }
 
+    @Test
```

Comment:
Nice. Let's add a similar test to the client module - it lacks meaningful tests.